scripts/lib.sh

```diff
@@ -91,8 +95,7 @@ sudo-askpass() {
 }
 
 upgrade-pip() {
-    grep -E '^(pip|setuptools|wheel)==' requirements-dev-frozen.txt |
-        xargs pip install --upgrade
+    pip-install $(grep -E '^(pip|setuptools|wheel)==' requirements-dev-frozen.txt)
```

Comment:
Let me see if I understand this correctly. One command extracts the indexes to use and the other one filters the 3 packages to pin. This all becomes a single command as per below.

Should the function be called something more clarifying? (e.g. `install-pinned-pip-tools`)

```
❯ pip install $(grep ^-- requirements-base.txt) $(grep -E '^(pip|setuptools|wheel)==' requirements-dev-frozen.txt)
Looking in indexes: https://pypi.devinfra.sentry.io/simple, https://pypi.org/simple
Requirement already satisfied: wheel==0.37.1 in ./.venv/lib/python3.8/site-packages (0.37.1)
Requirement already satisfied: pip==22.1.2 in ./.venv/lib/python3.8/site-packages (22.1.2)
Requirement already satisfied: setuptools==56.0.0 in ./.venv/lib/python3.8/site-packages (56.0.0)
```

Reply:
yep!  -- had to change the syntax here a little bit because you can't xargs a function